bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -136,6 +144,27 @@ <h2 class="go">{{ _('Download Firefox') }}</h2>
             <a class="fxos-link" href="{{ url('firefox.desktop.index') }}">{{ _('Learn more') }}</a>
           </div>
         </li>
+      {% if waffle.switch('mozorg-home-privacy-day') and LANG.startswith('en') %}
+        <li id="promo-6" class="item promo-large-landscape privacy-day" tabindex="0" data-name="January 28 is Data Privacy Day">
+          <h2 class="primary go">{% if LANG == 'en-US' %}January 28{% else %}28 January{% endif %} is Data Privacy Day</h2>
```

Comment:
I may be overthinking this bit. We don't localize this page for en-ZA or en-GB do we?